modules/ROOT/pages/index.adoc

```diff
@@ -82,6 +82,134 @@ Monitor APIs and endpoints at scheduled intervals. Receive reports about whether
 
 See xref:api-functional-monitoring::afm-in-anypoint-platform.adoc[API Functional Monitoring in Anypoint Monitoring].
 
+== Anypoint Monitoring Features
+
+Anypoint Monitoring provides many features to Gold and Platinum subscribers. However, for more comprehensive monitoring services, Titanium subscribers receive additional charts and functionality.
+
+|===
+| Feature | Gold/Platinum Support | Titanium Support
```

Comment:
Gold / Plat / Titanium are subscription tiers. Let's standardize with the rest of the doc on how we talk about these tiers. Let's not introduce "Support" when describing tiers.